Multisig/UI/Safe Management/Create Safe/GoogleWeb3AuthLoginModel.swift

```diff
@@ -22,29 +22,35 @@ class GoogleWeb3AuthLoginModel {
 
     func loginWithCustomAuth() {
         Task {
-            let sub = SubVerifierDetails(loginType: .installed,
-                                         loginProvider: .google,
-                                         clientId: App.configuration.web3auth.googleClientId,
-                                         verifier: App.configuration.web3auth.googleVerifier,
-                                         redirectURL: App.configuration.web3auth.redirectScheme + GoogleWeb3AuthLoginModel.schemePostfix
-            )
-            let tdsdk = CustomAuth(aggregateVerifierType: .singleLogin,
-                                   aggregateVerifier: App.configuration.web3auth.googleVerifier,
-                                   subVerifierDetails: [sub],
-                                   network: .CYAN
-            )
-            await MainActor.run(body: {
-                authorizationComplete()
-            })
-
-            let data = try await tdsdk.triggerLogin(browserType: .asWebAuthSession)
-            await MainActor.run(body: {
-                let key = data["privateKey"] as? String
-                let userInfo = data["userInfo"] as? Dictionary ?? [:] as Dictionary
-                let email = userInfo["email"] as? String ?? "email withheld"
-
-                keyGenerationComplete(key!, email)
-            })
+            do {
+                let sub = SubVerifierDetails(loginType: .installed,
+                                             loginProvider: .google,
+                                             clientId: App.configuration.web3auth.googleClientId,
+                                             verifier: App.configuration.web3auth.googleVerifier,
+                                             redirectURL: App.configuration.web3auth.redirectScheme + GoogleWeb3AuthLoginModel.schemePostfix
+                )
+                let tdsdk = CustomAuth(aggregateVerifierType: .singleLogin,
+                                       aggregateVerifier: App.configuration.web3auth.googleVerifier,
+                                       subVerifierDetails: [sub],
+                                       network: .CYAN
+                )
+                await MainActor.run(body: {
+                    authorizationComplete()
+                })
+                let data = try await tdsdk.triggerLogin(browserType: .asWebAuthSession)
```

Comment:
triggerLogin() is declared as throwing. But the print in the catch clause (line 52) is never reached.